File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/Java/5_LinkedList.js
public class LinkedListInJava {
    public static class ListNode{
        int val; // mang g.tri value ma minh muon the hien
        ListNode next; //con tro next
        ListNode() { } //1. Ham tao defaut
        //2. Ham co tham so val
        ListNode(int val){
            this.val = val;
        }
    }

    public static void main(String[] args) {
        //Tao ra 3 node
        ListNode n1 = new ListNode(1);
        ListNode n2 = new ListNode(2);
        ListNode n3 = new ListNode(3);

        //noi cac node lai voi nhau
        n1.next = n2;
        n2.next = n3;

        System.out.print("Linked List: ");
        //duyet cac p.tu trong linked list
        ListNode pHead = n1;
        while(pHead != null){
            System.out.print(pHead.val + " ");
            pHead=pHead.next;
        }
    }
}

File: THE_BROWN_BOX/00_LEARN1_CTDL & GIAI THUAT - BASIC/C++/5_LinkedList.cpp
#include <iostream>
using namespace std;

struct ListNode
{
    int val; // mang g.tri value ma minh muon the hien
    ListNode* next; //con tro next
    //1. Ham tao ko co tham so
    ListNode()
    {
        val = 0;
        next = nullptr;
    }
    //2. Ham tao cos tham so minh muon truyen vao
    ListNode(int x)
    {
        val = x;
        next = nullptr;
    }
};
int main()
{
    //Tao ra 3 node
    ListNode* n1 = new ListNode(1);
    ListNode* n2 = new ListNode(2);
    ListNode* n3 = new ListNode(3);

    //noi cac node lai voi nhau
    n1->next = n2;
    n2->next = n3;

    cout<< "Linked List: ";
    //duyet cac p.tu trong linked list
    ListNode* pHead = n1;
    while(pHead != nullptr){
        cout<< pHead->val <<" ";
        pHead=pHead->next;
    }
		//vi sd toan tu new nen can delete no di
    delete n1;
    delete n2;
    delete n3;
}